examples/trpo_gym_cartpole.py

```diff
@@ -22,7 +24,6 @@ def run_task(*_):
         policy=policy,
         baseline=baseline,
         batch_size=4000,
-        max_path_length=env.horizon,
```

Comment:
why not add `from garage.misc.util import horizon`, then `max_path_length=horizon(env)`?

Reply:
I didn't know that horizon was moved there. Thanks for letting me know Chang, will do.

Reply:
Np. Could you also add this in examples/trpo_gym_pendulum.py, please? It has the same problem. Thanks.

Reply:
Ok, will do!